### PR TITLE
build-bottle-pr: no more formula options, and we don't use Circle anymore

### PR DESCRIPTION
**Pull request by issyl0**
- Remove CircleCI `--keep-old` test-bot config as we're on Azure mostly
  now.
- Don't tell users they should `brew upgrade --devel hub` as options are
  deprecated.

### CODE REVIEW

File: cmd/brew-build-bottle-pr.rb [hunk 1]
```diff
@@ -99,7 +90,7 @@ def hub_pull_request(formula, remote, branch, message)
     if hub_version >= Version.new("2.3.0")
       args += ["-a", ENV["HOMEBREW_GITHUB_USER"] || ENV["USER"], "-l", "bottle"]
     else
-      opoo "Please upgrade hub\n  brew upgrade --devel hub"
+      opoo "Please upgrade hub\n  brew upgrade hub"
```
Comment (maxim-belkin) [inline cmd/brew-build-bottle-pr.rb]: Not a requirement but (if you've got time) -- I think it would be nest to submit this change in a separate PR.

**Comment by issyl0** [comment]
Confirmed with @maxim-belkin that removing the CircleCI stuff wouldn't affect the Linuxbrew/xorg workflows.

File: cmd/brew-build-bottle-pr.rb (hunk 1 above)
Comment (issyl0) [reply cmd/brew-build-bottle-pr.rb]: Sorry but I don't have time. If this were a bigger change, I would, but it's so small.
